.gitignore

```diff
@@ -7,3 +7,7 @@ bundle.*
 **/dist
 */**/LICENSE
 **/*.tgz
+
+# system files
+.DS_Store
+*/.DS_Store
```

Comment:
Is this line necessary? On my machine, git seems to treat `.DS_Store` from the line above as "ignore this filename anywhere in the tree".

Also, this is a weird pedantic thing on my part, but could you make sure this file ends in a newline?